src/lib/providers/usernamePasswordProvider.ts

```diff
@@ -0,0 +1,136 @@
+import { compareSync, hashSync } from 'bcrypt';
+import { AuthistOptions, User, UserCredentials, UserInfo } from '../authist';
+import { createCredentials } from '../credentialsService';
+import { ERROR_CODE, NotAuthenticated } from '../error';
+import { handleError } from '../providerUtils';
+
+export enum HashingAlgorithm {
+    Bcrypt = 'bcrypt',
```

Comment:
Is it possible to use any Node.js' built in hashing algs https://nodejs.org/api/crypto.html#crypto_crypto_gethashes ?

Reply:
i think it is possible but `bcrypt` for example is not part of them. I am not sure about the implementation of the check password function but i can try or we can create an issue and try it later